test/PProf.jl

```diff
@@ -64,10 +64,10 @@ end
     end
     for i in 1:2
         if i == 1
-            data = Profile.fetch()
+            data = Profile.fetch(include_meta = true)
             args = (data,)
```

Comment:
I guess we should continue to support the old interface where you don't set `include_meta = true`?
Or would you rather require that (and assert if it's missing) and make this a breaking version change update?

Reply:
Okay I pushed up a commit, here: [7b1f044](https://github.com/JuliaPerf/PProf.jl/pull/40/commits/7b1f04406471789fb75e2cc3c79da03099edab49), for supporting non-meta profiles, but it adds a lot of complexity and i'm not sure it's right, so i'd probably just rather always require it for now. We can always relax this again in the future if we want to?

Reply:
I reverted that commit, and i think this looks good to go

Reply:
K never mind, i ended up supporting backwards compatibility. It wasn't too bad. 👍